eclass/cuda.eclass

```diff
@@ -81,7 +81,7 @@ cuda_gccdir() {
 
 	if [[ -n ${gcc_bindir} ]]; then
 		if [[ -n ${flag} ]]; then
-			ret="${flag}\"${gcc_bindir}\""
+			ret="${flag}${gcc_bindir}"
```

Comment:
While I don't really mind this, I'd really prefer if we tried to determine the root cause for failure here. Especially that later on additional inline quotes are added into `NVCCFLAGS` (compiler_options).